coalib/misc/Future.py

```diff
@@ -0,0 +1,83 @@
+from functools import partial
+
+
+# Start ignoring all
+
+# FIXME: Remove `partialmethod` when dropping Python 3.3 support.
```

Comment:
shouldn't you like import it when on py 3.4 or higher and only use your implementation in the other cases? Bugfixes might happen.